### ISSUE TITLE
Issues with Selecting Timeslot

### ISSUE THREAD

**Issue by korkny** [yilmaznaslan/berlin-auslaenderbehorde-termin-bot#46]
Hi there, 

Thanks so much for sharing this project! This auslaenderbehoerde is the worst of the worst.

I'm v much an amateur here so forgive me. I'm running the program and all seems to be well except for when it reaches the date selection point. At that point I get this info. 

2023-05-10 15:55:47 INFO Section3DateSelectionHandler There are 2 available timeslots
2023-05-10 15:55:47 INFO Section3DateSelectionHandler Timeslot: 0, Value: Please select
2023-05-10 15:55:47 INFO Section3DateSelectionHandler Timeslot: 1, Value: 
2023-05-10 15:55:47 ERROR TerminFinder Exception occurred during the process, quitting.
org.example.exceptions.FormValidationFailed: Validating the time slots has failed
        at org.example.formhandlers.Section3DateSelectionHandler.handleTimeSelection(Section3DateSelectionHandler.java:87) ~[main/:?]
        at org.example.formhandlers.Section3DateSelectionHandler.fillAndSendForm(Section3DateSelectionHandler.java:41) ~[main/:?]
        at org.example.TerminFinder.fillAndSendFormWithExceptionHandling(TerminFinder.java:116) ~[main/:?]
        at org.example.TerminFinder.run(TerminFinder.java:87) ~[main/:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:577) ~[?:?]
        at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358) ~[?:?]
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
        at java.lang.Thread.run(Thread.java:1623) [?:?]
2023-05-10 15:55:47 INFO IoUtils File name :exception_2023-05-10_15:55:47_exception.html, exception_2023-05-10_15:55:47_exception.png

Basically I'm pretty sure it keeps coming across a faulty drop down selection which just says 'please select' - as I have gotten a few times before when going through the site manually.

Since this happens pretty often, is there a way to have this written into the project with some fail safe for if the value is not a time? (ie. please select). 

Going to leave the program running anyway, in case it is just a problem with the webpage and a proper appointment rears its head.

**Comment by yilmaznaslan** [comment]
Yeah that's true. Sometimes even though calender page is opened, time slots are not shown. Although it is not a blocker for the execution of the next search, I will make logs the prettiest so that it doesn't look scary ;)

**Comment by yilmaznaslan** [comment]
Can you please tell me what is commit hash that you are currently running the application? Simply `git show` like below 
![image](https://github.com/yilmaznaslan/berlin-auslaenderbehorde-termin-bot/assets/58464107/c31a53c1-be7e-4fe9-996c-896abb028a55)

**Comment by allentiak** [comment]
It fails due to a captcha.
Where should I find the log?

**Comment by yilmaznaslan** [comment]
Bot is not able select the time slot automatically due to captcha. It used to work in the past
